src/test/scala/cornichonTests/FeatureWithProduct.scala

```diff
@@ -0,0 +1,119 @@
+package cornichonTests
+
+import com.github.agourlay.cornichon.CornichonFeature
+
+trait FeatureWithProduct {
+  this: CornichonFeature =>
+
+  def AddProduct =
+    Attach {
+      addProductType
+      addProduct
+    }
+
+  def DeleteProduct =
```

Comment:
```suggestion
  def deleteProduct =
```
to be consistent?

Reply:
Will rename. There're methods with the same names within the trait, so I thought to leave them uppercase to emphasise that they are external